utils/addKeyboardListrener/addKeyboardListrener.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+// eslint-disable-next-line simple-import-sort/imports
+import { useEffect, useState, KeyboardEvent, KeyboardEventHandler } from 'react';
+
+export const addKeyboardListener = (targetKey: string): boolean => {
```

Comment:
you just renamed it, but it is hook. Do NOT use hook, make it utility addKeyboardListener(target, callback) {
  if (pressedKey == target) {
    callback(event);
  }
}